LayoutTests/imported/w3c/web-platform-tests/css/css-scoping/host-not-001.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>:host combined with :not</title>
+<link rel="help" href="https://drafts.csswg.org/selectors/#featureless">
+<link rel="match" href="/css/reference/ref-filled-green-100px-square-only.html">
+<p>Test passes if there is a filled green square.</p>
+<div id=host>
+</div>
+<script>
+  host.attachShadow({mode: "open"}).innerHTML = `
+    <style>
+      :host {
+        width: 100px;
+        height: 100px;
+        background-color: green;
+      }
+      :not(div):host {
+        background-color: red;
+      }
```

Comment:
Another good testcase is `:host:not(:hover)`

Reply:
Added